manifest_test.go

```diff
@@ -867,6 +868,16 @@ func (m *ManifestTestSuite) TestManifestEntryBuilder() {
 	m.Assert().Equal(0, *data.SortOrderID())
 }
 
+func (m *ManifestTestSuite) TestManifestWriterMeta() {
+	sch := NewSchema(0, NestedField{ID: 0, Name: "test01", Type: StringType{}})
+	w, err := NewManifestWriter(2, io.Discard, *UnpartitionedSpec, sch, 1)
+	m.Require().NoError(err)
+	md, err := w.meta()
+	m.Require().NoError(err)
+	m.Assert().NotEqual("null", string(md["partition-spec"]))
```

Comment:
you don't need the `Assert()` portion, you can just do `m.NotEqual` and `m.Equal`, the `m.Assert()` part is a convenience for passing it around